webapp/.eslintrc.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
   },
   extends: [
     'standard',
-    'plugin:vue/recommended',
```

Comment:
@ulfgebhardt @Tirokk @mattwr18 @ogerly @Lulalaby I graduated here. I believe the two libraries are incomaptible since prettier supports vue linting.